src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -50,7 +50,15 @@
        HelixJobProperties- Must be JSON.  String describing Helix MC-specific metadata.       Default: <Blank>
        HelixArchLabel    - If HelixJobProperties is not set, we'll use this to fill it out    Default: <Blank>
        HelixConfigLabel  - If HelixJobProperties is not set, we'll use this to fill it out    Default: <Blank>
-       
+
+       **********************************************************************************************************************************
+       Re-queuing Properties:
+       **********************************************************************************************************************************
+       UseContinuationRunner  - Use runner script that is capable of sending to another queue when finished  Default: False
+       SecondaryQueue         - When submitting multi-stage tests, queue for secondary jobs                  Default: <Blank>
```

Comment:
I think I would use the term Coninuation everywhere here instead of Secondary in some places.